### PR TITLE
SW-4906 Implement router-specific deliverable views so we can encapsulate the view-agnostic components

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2215?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* **#2215** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2215?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2214** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2214?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2212** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2212?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2215?utm_source=stack-comment).

File: src/scenes/AcceleratorRouter/DeliverableView.tsx [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-import { EditProps, ViewProps } from './types';
-import TitleBar from './TitleBar';
-import StatusBar from './StatusBar';
-import Description from './Description';
-import DocumentsUploader from './DocumentsUploader';
+import { EditProps, ViewProps } from 'src/components/DeliverableView/types';
+import TitleBar from 'src/components/DeliverableView/TitleBar';
+import StatusBar from 'src/components/DeliverableView/StatusBar';
+import Description from 'src/components/DeliverableView/Description';
+import DocumentsUploader from 'src/components/DeliverableView/DocumentsUploader';
```
Comment (nickgraz) [inline src/scenes/AcceleratorRouter/DeliverableView.tsx:12]: Eventually these might be replaced with view-specific components that wrap the shared component

File: src/scenes/DeliverablesRouter/DeliverableView.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (karthikbtf) [inline src/scenes/DeliverablesRouter/DeliverableView.tsx:1]: Should we merge this with the 'DeliverablesView' file in the same directory?
Or rename the other file to something else?

**Review by karthikbtf — APPROVED** [review]

**Comment by nickgraz** [comment]
### Merge activity

* **Feb 27, 4:27 PM EST**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2215).

**Comment by joshbuchea** [comment]
Sorry if those merge conflicts came from my PR for SW-4903

File: src/scenes/DeliverablesRouter/DeliverableView.tsx (hunk 2 above)
Comment (nickgraz) [reply src/scenes/DeliverablesRouter/DeliverableView.tsx:1]: I will rename it, I think something like `DeliverableViewWrapper` makes sense

**Pull request merged**